src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -352,10 +352,53 @@ public StudyEntity doGetStudyAndUpdateIsPrivate(UUID studyUuid, String headerUse
         return Mono.fromCallable(() -> studyInfosService.search(query)).flatMapMany(Flux::fromIterable);
     }
 
-    Flux<EquipmentInfos> searchEquipments(@NonNull UUID studyUuid, @NonNull String query) {
+    public static String escapeLucene(String s) {
```

Comment:
Why we do not use `QueryParser.escape`

Reply:
private ?

Reply:
May be, but as it is static, it does not preclude any encapsulation nor pollute a namespace.